awx/ui_next/src/api/models/JobTemplates.js

```diff
@@ -44,6 +44,19 @@ class JobTemplates extends InstanceGroupsMixin(NotificationsMixin(Base)) {
   readCredentials(id, params) {
     return this.http.get(`${this.baseUrl}${id}/credentials/`, { params });
   }
+
+  associateCredentials(id, credential) {
+    return this.http.post(`${this.baseUrl}${id}/credentials/`, {
+      id: credential,
+    });
+  }
+
+  disassociateCredentials(id, credential) {
+    return this.http.post(`${this.baseUrl}${id}/credentials/`, {
+      id: credential,
```

Comment:
Calling this variable `credentialId` instead of `credential` would be bit more clear